src/Symfony/Component/Security/Http/EventListener/IsGrantedAttributeListener.php

```diff
@@ -46,7 +50,15 @@ public function onKernelControllerArguments(ControllerArgumentsEvent $event)
             $subject = null;
 
             if ($subjectRef) {
-                if (\is_array($subjectRef)) {
+                if ($subjectRef instanceof Expression) {
+                    $this->expressionLanguage ??= class_exists(Expression::class)
```

Comment:
the class_exists check is not needed to me because $subjectRef is already an instance of `Expression`, which means it is loaded

Reply:
Good point, didn't think of that, removed